project/app/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,8 @@
     <string name="preferencesAutostart">Autostart</string>
     <string name="preferencesRemoteCommand">Remote commands</string>
     <string name="preferencesRemoteCommandSummary">Enable remote commands</string>
+    <string name="preferencesGeocode">Geocode</string>
+    <string name="preferencesGeocodeSummary">Enable geocoding</string>
```

Comment:
I can contribute German translations for this:

```xml
    <string name="preferencesGeocode">Geocodierung</string>
    <string name="preferencesGeocodeSummary">Geocodierung aktivieren</string>
```

this follows the translations for e.g. remote commands:

https://github.com/owntracks/android/blob/d2e1c22c927927e9dfd8f64a8171188e4d06ffa9/project/app/src/main/res/values-de/strings.xml#L36-L37